### PR TITLE
AvalancheGo V1.9.12 Release Notes.

### PR DESCRIPTION
**Pull request by cgcardona**

### CODE REVIEW

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 1]
```diff
@@ -6,6 +6,149 @@
...
 
+## V1.9.12 [View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12)
+
+**Banff.12 - Cortina Preparations**
+
+This version is backwards compatible to
+[v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0).
+It is optional, but encouraged. The supported plugin version is `24`.
+
+### Networking
```
Comment (StephenButtolph) [inline docs/apis/avalanchego/avalanchego-release-notes.md:17]: I think rather than `### XXX` titles we use `**XXX**` in the rest of this file

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 2]
```diff
@@ -6,6 +6,149 @@
...
+- Added P-chain `RemoveSubnetValidatorTx` example using the wallet
+- Added X-chain `CreateAssetTx` example using the wallet
+
+### Configs
+
+- Added support to specify `HTTP` server timeouts
+  - `--http-read-timeout`
+  - `--http-read-header-timeout`
+  - `--http-write-timeout`
+  - `--http-idle-timeout`
```
Comment (StephenButtolph) [inline docs/apis/avalanchego/avalanchego-release-notes.md:89]: We should probably add these to the node configs (either in this PR or a follow up)

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 3]
```diff
@@ -6,6 +6,149 @@
...
+
+### APIs
+
+- Added `avm` block APIs
+  - `avm.getBlock`
+  - `avm.getBlockByHeight`
+  - `avm.getHeight`
```
Comment (StephenButtolph) [inline docs/apis/avalanchego/avalanchego-release-notes.md:96]: We'll probably want to document these (either in this PR or a followup)

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 4]
```diff
@@ -6,6 +6,149 @@
...
+
+### Configs
+
+- Added support to specify `HTTP` server timeouts
+  - `--http-read-timeout`
+  - `--http-read-header-timeout`
+  - `--http-write-timeout`
+  - `--http-idle-timeout`
+
+### APIs
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:91]: We would need to update the developer doc on these APIs. However, that can be a separate PR.

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 5]
```diff
@@ -6,6 +6,149 @@
...
+  - `platform.addSubnetValidator`
+  - `platform.createSubnet`
+  - `platform.exportAVAX`
+  - `platform.importAVAX`
+  - `platform.createBlockchain`
+  - `platform.getBlockchains`
+  - `platform.getStake`
+  - `platform.getMaxStakeAmount`
+  - `platform.getRewardUTXOs`
+- Deprecated the `stake` field in the `platform.getTotalStake` response in favor of `weight`
```
Comment (StephenButtolph) [inline docs/apis/avalanchego/avalanchego-release-notes.md:150]: Not sure how we express deprecations in the docs normally

File: docs/apis/avalanchego/avalanchego-release-notes.md (hunk 1 above)
Comment (yulin-dong) [reply docs/apis/avalanchego/avalanchego-release-notes.md:17]: Yes, otherwise, they become separate titles.

Comment (cgcardona) [reply docs/apis/avalanchego/avalanchego-release-notes.md]: Confirmed and fixed.

**Review by yulin-dong — APPROVED** [review]
